openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1088,32 +1089,22 @@ Deferred Credential Response MUST be sent using the `application/json` media typ
 
 ## Deferred Credential Error Response {#deferred-credential_error_response}
 
-When the Deferred Credential Request is invalid or unauthorized, or the credential is not available yet, the Credential Issuer constructs the error response as defined in this section.
-
-The following additional clarifications are provided for the following parameters already defined in section 3.1 of [@!RFC6750]:
-
-`invalid_request`:
-
-- Credential Request was malformed, e.g., the parameter `transaction_id` is missing or malformed.
-
-`invalid_token`:
-
-- Credential Request contains the wrong Access Token or the Access Token is missing.
+When the Deferred Credential Request is invalid or the credential is not available yet, the Credential Issuer constructs the error response as defined in (#credential-error-response).
```

Comment:
```suggestion
When the Deferred Credential Request is invalid or the credential is not available yet, the Credential Issuer SHOULD use the error response as defined in (#credential-error-response).
```

Reply:
why SHOULD? this feels like a MUST, also for consistency
```suggestion
When the Deferred Credential Request is invalid or the credential is not available yet, the Credential Issuer MUST construct the error response as defined in (#credential-error-response).
```